app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd = command.split()
+        command_name = cmd[0]
+        source_name = cmd[1]
+        copy_name = cmd[2]
```

Comment:
You should use unpacking
```suggestion
        command_name, source_name, copy_name = cmd
```